src/databaseConnection.ts

```diff
@@ -51,14 +47,45 @@ export async function databaseInit(): Promise<void> {
           game_round BIGINT DEFAULT 0 NOT NULL,
           starting_funds BIGINT NOT NULL,
           small_blind BIGINT NOT NULL,
-          small_blind_who SERIAL REFERENCES Players(id) NOT NULL,
+          small_blind_who VARCHAR(250) REFERENCES Players(token) NOT NULL,
           current_table_value BIGINT,
-          current_player SERIAL REFERENCES Players(id) NOT NULL
+          current_player VARCHAR(250) REFERENCES Players(token) NOT NULL
         )`
-      )
-    } catch (err) {
-      console.error(err)
-    }
+    )
+
+    await client.query(
+      `create or replace function insert_with_random_key (
+            game_master text,
```

Comment:
I think we should try using same style for database queries, for example type name was previously written with capital letters.